articles/connections/passing-parameters-to-idps.md

```diff
@@ -0,0 +1,127 @@
+
+As an example, let's use WordPress, which allows you to pass an optional `blog` parameter to its OAuth 2.0 authorization endpoint (for more information, see [WordPress's OAuth 2.0 documentation](https://developer.wordpress.com/docs/oauth2/)). Let's assume that you want to always request that users have access to the `myblog.wordpress.com` blog when logging in using WordPress.
+
+To accomplish this, you will map WordPress's `blog` parameter to the existing accepted parameter of `access_type` and assign it a default value of `myblog.wordpress.com`:
+
+```
+{
+  "name": "WordPressConn",
+  "strategy": "wordpress",
+  "options": {
+    "upstream_params": {
+      "blog": {
+        "alias": "access_type",
+        "value": "myblog.wordpress.com"
+      }
+    }
+  }
+}
+```
+
+```har
+{
+  "method": "PATCH",
+  "url": "https://${account.namespace}/api/v2/connections/CONNECTION-ID",
+  "headers": [
+    { "name": "Content-Type", "value": "application/json" }
+  ],
+  "postData": {
+    "mimeType": "application/json",
+    "text": "{ 
+      \"options\": { 
+        \"client_id\": \"...\", 
+        \"client_secret\": \"...\", 
+        \"upstream_params\": { 
+          \"blog\": { 
+            \"alias\": \"access_type\",
+            \"value\": \"myblog.wordpress.com\"
+          }
+        }, 
+        \"authorizationURL\": \"https://public-api.wordpress.com/oauth2/authorize\", 
+        \"tokenURL\": \"https://public-api.wordpress.com/oauth2/token\",
+        \"scope\": \"auth\" 
+      } 
+    }"
+  }
+}
+```
+
+You can test out your code and see other available attributes in our [Management API Explorer](https://auth0.com/docs/api/management/v2#!/Connections/post_connections).
+
+
+## Passing Dynamic Parameters
+
+Sometimes you may want to configure parameters per user. To do this, use the `upstream_params` element of the `options` attribute to specify a mapping between one of the existing accepted parameters to the parameter accepted by the Identity Provider. Then, when authorizing the user, dynamically add parameters to the authorization query.
+
+As an example, let's use Twitter, which allows you to pass an optional `screen_name` parameter to its OAuth authorization endpoint (for more information, see [Twitter's API reference](https://developer.twitter.com/en/docs/basics/authentication/api-reference/authorize)). Twitter's `screen_name` parameter pre-fills the username input box of the login screen with the given value, so we want to map it to the existing accepted parameter of `login_hint`.
+
+```
+{
+  "name": "TwitterConn",
+  "strategy": "twitter",
+  "options": {
+    "upstream_params": {
+      "screen_name": {
+        "alias": "login_hint"
+      }
+    }
+  }
+}
+```
+
+```har
+{
+  "method": "PATCH",
+  "url": "https://${account.namespace}/api/v2/connections/CONNECTION-ID",
+  "headers": [
+    { "name": "Content-Type", "value": "application/json" }
```

Comment:
looks like you are missing the `Bearer` header (where the token goes, that authenticates with the API).

I think the headers should be like that:
```
"headers": [{
    "name": "Authorization",
    "value": "Bearer YOUR_ACCESS_TOKEN"
  }, {
    "name": "Content-Type",
    "value": "application/json"
  }],
```